### ISSUE TITLE
Investigate different snapshot configurations

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#3758]
## Goal:

 * understand the implications of different snapshot settings
 * open issues if there are any
 * document for each setting expected state/implications (maybe in docs?)

## Prerequiste

 - [X] we can configure snapshot configs in our benchmarks
 - [X] we have metrics for snapshot available
 - [ ] we have a stable benchmark which defines our steady state (how the system should look like normally)

## Procedure

 * create an separate issue for the specific configuration you want to test
 * write down the expected state and experiment configurations
 * link to this issue
 * run experiment as described below
 * document the actual state - compare to steady state
 * create new issues if observed weird/unexpected behavior

## Experiment

 * create new namespace/benchmark
* change specific snapshot setting
 * default setup, which means:
   * 3 brokers
   * 3 partitions
   * replication factor 3
   * 300 wfi/s - starter
   * 12 worker 
   * elastic exporter
 * run time ~24 h

**Comment by Zelldon** [comment]
We stopped working on this after #3760
